<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDrawBoundaryCard</class>
 <widget class="QWidget" name="FormDrawBoundaryCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Colores primarios*/
/* 
gris oscuro #222222 #333333 #444444
gris claro #999999 #DDDDDD
verde claro #C8CC8E
vinotinto #742427
*/
/*Colores secundarios */
/* 
Azules #36C9C6 #00BDB9 #77ACA2
rojos #910D3F #C70039 #F94646
naranjas #D34E24 #F28123 #F7F052
*/

/* Funetes/*

/*
italic: font: italic 9pt &quot;Ubuntu&quot;;
regular: font: 9pt &quot;Ubuntu&quot;;
light: font: 300 9pt &quot;Ubuntu&quot;;
medium: font: 500 9pt &quot;Ubuntu&quot;;
bold: font: 700 9pt &quot;Ubuntu&quot;;
*/

/****************************************************************************************************************************************************/


QLabel#label_cardNameBoundary{
margin-left: 10px;
}



QFrame#frame_card{
background: #222222;
border-radius:4px
}

QFrame[QFrameStyle_=&quot;1&quot;]{
background: transparent;
}

QFrame[QFrameStyle_=&quot;2&quot;]{
background: transparent;
border-radius:5px;
border: 1px solid #F94646;
}

QToolButton[style_material_point_card_button=&quot;1&quot;]{
background-color: transparent;
border: 1px solid #222222;
border-radius: 3px ;
}

QToolButton[style_material_point_card_button=&quot;1&quot;]:hover{ 
background-color: #444444;
}

QToolButton[style_material_point_card_button=&quot;1&quot;]:pressed{
border-top: 2px solid #222222;
border-left: 2px solid #222222;
}  


QLineEdit#lineEdit_nameBoundary{
font: 9pt &quot;Ubuntu&quot;;
color: #DDDDDD;
background-color: #444444;
border-radius: 2px ;
padding-right: 6px;
padding-left: 6px;

}



QFrame#frame_edit{
background: #333333;
border: 1px solid #222222;
border-radius: 5px
}




/*◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘           QLabel           ◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘*/

QLabel[QLabelStyle=&quot;1&quot;] {
font: 700 9pt &quot;Ubuntu&quot;;
color: #DDDDDD;
}

QLabel[QLabelStyle=&quot;2&quot;] {
font: 500 9pt &quot;Ubuntu&quot;;
color: #DDDDDD;
}

QLabel[QLabelStyle=&quot;3&quot;] {
font: 300 9pt &quot;Ubuntu&quot;;
color: #DDDDDD;
}



/*◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘           QComboBox            ◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘*/
QComboBox[QComboBoxStyle=&quot;1&quot;] {
    border: none;
    background-color: #444444;
	color: #DDDDDD;
    border-radius: 2px;
    padding: 5px;
	font:  9pt &quot;Ubuntu&quot;;
    selection-background-color: #808080;
}
QComboBox[QComboBoxStyle=&quot;1&quot;] QAbstractItemView {
    border: none;
    background-color: #404040;
    color: white;
    selection-background-color: #808080;
    font: 700 9pt &quot;Ubuntu&quot;;
}



QComboBox[QComboBoxStyle=&quot;2&quot;] {
    border: none;
    background-color: transparent;
	color: #DDDDDD;
    border-radius: 2px;
    padding: 5px;
	font:  9pt &quot;Ubuntu&quot;;
    selection-background-color: #808080;
}

QComboBox[QComboBoxStyle=&quot;2&quot;]::down-arrow {
    /* Para ocultar la flecha de selección */
    width: 0;
    height: 0;
    padding: 0;
    border: none;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_card">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_Tx">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="QFrameStyle_" stdset="0">
         <number>2</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_BoundaryTx">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Tx</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="QLabelStyle" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_Ty">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="QFrameStyle_" stdset="0">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_BoundaryTy">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Ty</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="QLabelStyle" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cardNameBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Material. Point 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="QLabelStyle" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_nameBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_okBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../recursos/iconos/iconos_generales/ok.svg</normaloff>../recursos/iconos/iconos_generales/ok.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="style_material_point_card_button" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_exitBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../recursos/iconos/iconos_generales/exit_2.svg</normaloff>../recursos/iconos/iconos_generales/exit_2.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="style_material_point_card_button" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_editBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../recursos/iconos/iconos_generales/edit.svg</normaloff>../recursos/iconos/iconos_generales/edit.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="style_material_point_card_button" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_showHideBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>25</horstretch>
          <verstretch>25</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../recursos/iconos/iconos_menu_draw_mesh/view.svg</normaloff>../recursos/iconos/iconos_menu_draw_mesh/view.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="style_material_point_card_button" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_closeBoundary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../recursos/iconos/iconos_consola/exit.svg</normaloff>../recursos/iconos/iconos_consola/exit.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="style_material_point_card_button" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
